frontend/src/component/personalDashboard/ConnectSDK.tsx

```diff
@@ -52,7 +52,7 @@ export const CreateFlag: FC<{ project: string }> = ({ project }) => {
                 Create a feature flag
             </TitleContainer>
             <div>
-                <p>The project currently holds no feature toggles.</p>
+                <p>The project currently holds no feature flags.</p>
                 <p>Create a feature flag to get started.</p>
```

Comment:
```suggestion
                <p>Create one get started.</p>
```

Reply:
Whoops, you're right; missed this and was too eager on the auto-merge. I'll do this in a followup